android/gradle/libs.versions.toml

```diff
@@ -3,6 +3,7 @@ agp = "8.5.2"
 kotlin = "2.0.0"
 cargo-ndk = "0.3.4"
 ktfmt = "0.19.0"
+lifecycleRuntimeComposeAndroid = "2.8.4"
```

Comment:
I think our recent PRs crossed and I cleaned up the same things. This is a duplicate of `androidx-lifecycle` that's not used anymore I think.

```suggestion
```